client.go

```diff
@@ -162,7 +165,8 @@ func (c *Client) Do(ctx context.Context, req *http.Request) (resp *http.Response
 
 	// If 401, get new token and retry one time.
 	if resp.StatusCode == http.StatusUnauthorized {
-		tok, tokErr := c.Token.Create(ctx, &TokenCreateRequest{Username: c.username, Password: c.password})
+		tcr := &TokenCreateRequest{AuthMethod: c.authMethod, Username: c.username, Password: c.password, AppCredID: c.appCredID, AppCredSecret: c.appCredSecret}
```

Comment:
Then you have to set other information inside `c.Token.Create`

Reply:
what information?

Reply:
https://github.com/bizflycloud/gobizfly/blob/aa1015fb9f07ee74ad943c18e960ab10b774e012/token.go#L56

Reply:
Ah. I see. Thanks